File: TS/3.ts
function countEven(num: number): number {
    let counter = 0;
    while (num !== 0) {
        const n = num % 10;
        if (n % 2 === 0) counter++;
        num = Math.floor(num / 10);
    }
    return counter;
}

const readline = require('readline').createInterface({
    input: process.stdin,
    output: process.stdout
});

readline.question('', (n: string) => {
    let count = 0;
    const processNumber = () => {
        if (count >= parseInt(n)) {
            readline.close();
            return;
        }
        readline.question('', (tmp: string) => {
            console.log(countEven(parseInt(tmp)));
            count++;
            processNumber();
        });
    };
    processNumber();
});

File: Go/3.go
package main

import "fmt"

func countEven(number int) int {
	counter := 0
	for number != 0 {
		n := number % 10
		if n % 2 == 0 && n != 0 {
			counter++
		}
		number /= 10
	}
	return counter
}

func main() {
	var n int
	fmt.Scanln(&n)

	for _ = range n {
		var num int
		fmt.Scanln(&num)
		fmt.Println(countEven(num))
	}
}
